catalog-rest-service/src/main/resources/json/schema/entity/services/databaseService.json

```diff
@@ -193,6 +197,9 @@
             },
             {
               "$ref": "./connections/database/sampleDataConnection.json"
+            },
+            {
+              "$ref": "./connections/database/pinotDBConnection.json"
```

Comment:
This reference is failing due to the filename. The filename should be `pinotDBConnection.json`.